### ISSUE TITLE
Having a way to half fullscreen an application

### ISSUE THREAD

**Issue by Zaxiure** [hyprwm/Hyprland#7720]
### Description

Coming from yabai this is the only functionality I feel like I'm really missing, it has an option to zoom the parent node. The video below is showcasing what I'm talking about.

https://github.com/user-attachments/assets/8fcc543c-d8b5-4c4c-9410-f3b64320d731

**Comment by 299792458ms** [comment]
There is the option to make windows fullscreen respecting gaps & restricted areas(bars), its not the same but hopefully its of some use.

```
bind = $mod, F, fullscreen, 1    # semifullscreen
bind = $mod Shift, F, fullscreen, 0    # regular fullscreen
```

**Comment by Zaxiure** [comment]
Yeah, I'm already using that feature. I see this as a completely seperate thing.

**Comment by Vladimir-csp** [comment]
Might be similar to/covered by #3575

**Comment by 299792458ms** [comment]
> Might be similar to/covered by #3575

Pretty sure @Zaxiure just means to toggle a window to occupy the full size of the parent node and not *fake* virtual displays. But #3575 would be a super nice feature for big ass screens.

**Comment by vaxerski** [comment]
Hello there!

This issue has been closed, as we are moving from issues to discussions. More about this can be found [here](https://wiki.hyprland.org/Contributing-and-Debugging/Issue-Guidelines/) and in #9854.

Firstly, please make sure this issue is still relevant in latest Hyprland. If so, we ask you to open a [discussion](https://github.com/hyprwm/Hyprland/discussions) (please read the discussion guidelines first, in the pinned post)

Thank you for your understanding!
